README.md

```diff
@@ -463,6 +463,13 @@ Here are the [currently requested definitions](https://github.com/DefinitelyType
 
 If types are part of a web standard, they should be contributed to [TypeScript-DOM-lib-generator](https://github.com/Microsoft/TypeScript-DOM-lib-generator) so that they can become part of the default `lib.dom.d.ts`.
 
+#### What about type definitions with no matching package?
+
+If there's no matching Javascript code at all, for example if you're writing types for a spec, you should publish the types yourself, not on Definitely Typed. 
+`@types` packages are not meant to be imported directly. That is, you shouldn't create a Definitely Typed package that's meant to used like `import type { ... } from "@types/foo"`. Nor should you expect to write `import type { ... } from "foo"` when there's no `foo` installed.
+
+This is different from providing types for a browser-only Javascript library. There, the types are either resolved implicitly or using `/// <references types="foo" />`.
```

Comment:
I would also call out other "environment" packages, e.g. node, bun, and so on, which contribute globals that may not be imported.

But then again, _none_ of these are examples where there is no runtime element; they all describe values. Maybe that's the main thing; if a package doesn't contain any declarations that are reachable from value-space, then it shouldn't be on DT.

Reply:
> if a package doesn't contain any declarations that are reachable from value-space, then it shouldn't be on DT.

That is the main thing, but I can't figure out a jargon-free way to describe it.